lib/src/main/java/com/android/billingclient/api/BillingClientTesting.kt

```diff
@@ -75,7 +75,7 @@ class BillingClientTesting(context: Context, listener: PurchasesUpdatedListener)
                 val originalComponent = service.component
                 val intent = Intent(service)
                 intent.action = INTENT_STRING
-                intent.setPackage(INTENT_PKG)
+                intent.setPackage(packageName)
```

Comment:
Since the config app is now merged into the app that pulls in Register, the package name is now the same as the app that calls it. Trying to find the service based on the package name we define in the `app` module makes the implementing app crash because it can't find the service when we expect it to.

![image](https://user-images.githubusercontent.com/4405364/67813982-46f5a000-fa79-11e9-9263-f741e3764917.png)